com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -518,7 +518,10 @@ public void NetworkShow(ulong clientId)
                 return;
             }
             NetworkManager.SpawnManager.MarkObjectForShowingTo(this, clientId);
-            Observers.Add(clientId);
+            if (!Observers.Contains(clientId))
```

Comment:
This check isn't required since Observers is a HashSet.

Reply:
Oops, didn't mean to leave this in.

When trying to get clients able to send to other clients, they still have to stick to the observers list. I initially tried to do this by adding a `NetworkList<ulong> Observers` to every NetworkBehaviour and using NetworkVariable logic to handle updating it so that clients would be able to access the observer list as well, and this check was needed because NetworkList, unlike HashSet, does not handle duplicate values automatically.

That attempt broke a lot of tests, though, in ways that would be very hard to fix, so I abandoned that and just had the client send every id that could possibly match the target it is using, and had the server filter out anything that isn't in the observer list when forwarding the proxy message.

Will remove this check and look for any others I missed.